### DISCUSSION TITLE
Flasc circular import Error

### DISCUSSION THREAD

**Discussion by Ooussamaa** [NREL/flasc#88] [Q&A]
Hello,
I started working on Flasc and I am facing some issues; I can't run the existing scripts to see what are they made for, to customize them  after, this is due to this error: " circular import"

For exemple I tried to execute the command: "python layout_visualization.py" and I had this error: " ImportError: cannot import name 'dataframe_operations' from partially initialized module 'flasc' (most likely due to a circular import)" 
Can you please help me to fix it so I could start handling Flasc correctly !
Ps ( I am using Windows Powershell to execute my commands and Pyzo to edit the script )
Best regards
Oussama

**Comment by Bartdoekemeijer** [comment]
Hi, the issue is further explained in #49. To resolve the issue, please clone the repository manually and then `pip install` it. Directly installing it from PyPI will raise this error.
